docs/SUMMARY.md

```diff
@@ -7,74 +7,45 @@
-* [Exploring Ethereum](tutorials/exploring-ethereum/overview.md)
-  * [Mapping Blocks](tutorials/exploring-ethereum/map_block_meta_module.md)
-  * [Filtering Transactions](tutorials/exploring-ethereum/map_filter_transactions_module.md)
-  * [Retrieving Events of a Smart Contract](tutorials/exploring-ethereum/map_contract_events_module.md)
-* [Making eth_calls](tutorials/eth-calls.md)
-* [Rust](tutorials/rust/overview.md)
-  * [The Option struct](tutorials/rust/option.md)
-  * [The Result struct](tutorials/rust/result.md)
-
-## Developer's Guide
-
-* [Overview](developers-guide/overview.md)
-* [Dependency installation](developers-guide/installation-requirements.md)
-* [Manifest](developers-guide/creating-your-manifest.md)
-* [Protobuf schemas](developers-guide/creating-protobuf-schemas.md)
-* [Modules](developers-guide/modules/README.md)
-  * [Module types](developers-guide/modules/types.md)
-  * [Inputs](developers-guide/modules/inputs.md)
-  * [Output](developers-guide/modules/outputs.md)
-  * [Module handlers](developers-guide/modules/setting-up-handlers.md)
-  * [Module handler creation](developers-guide/modules/writing-module-handlers.md)
-* [Running Substreams](developers-guide/running-substreams.md)
-* [GUI](developers-guide/gui.md)
-* [Parallel Execution](developers-guide/parallel-execution.md)
-* [Sink targets](developers-guide/sink-targets/README.md)
-  * [Substreams-powered subgraph](developers-guide/sink-targets/substreams-powered-subgraph.md)
-  * [Files](developers-guide/sink-targets/substreams-sink-files.md)
-  * [Key/value store](developers-guide/sink-targets/substreams-sink-kv.md)
-  * [SQL (Postgres, Clickhouse, others)](developers-guide/sink-targets/substreams-sink-sql.md)
-  * [Prometheus](developers-guide/sink-targets/substreams-sink-prometheus.md)
-  * [MongoDB](developers-guide/sink-targets/substreams-sink-mongodb.md)
-  * [Custom Sink Using JavaScript](developers-guide/sink-targets/custom-sink-js.md)
-* [Cookbook](developers-guide/cookbook.md)
-  * [Dynamic data sources](developers-guide/cookbook/dynamic-data-sources.md)
-  * [Factory parameterization](developers-guide/cookbook/factory-params.md)
-  * [Advanced parameters](developers-guide/cookbook/advanced-params.md)
-  * [Keys in stores](developers-guide/cookbook/keys-in-stores.md)
-  * [Aggregation windows](developers-guide/cookbook/aggregation-windows.md)
-  * [Subgraph entity changes](developers-guide/cookbook/entity-changes.md)
 
 ## Reference & Specs
 
 * [Chains and endpoints](reference-and-specs/chains-and-endpoints.md)
 * [Substreams CLI reference](reference-and-specs/command-line-interface.md)
-* [Authentication](reference-and-specs/authentication.md)
-* [Manifests](reference-and-specs/manifests.md)
-* [Packages](reference-and-specs/packages.md)
-* [Rust crates](reference-and-specs/rust-crates.md)
+* [Manifests Reference](reference-and-specs/manifests.md)
+* [GUI Reference](reference-and-specs/manifests.md)
 * [Examples](reference-and-specs/examples.md)
```

Comment:
I'd try to make these examples more visible, maybe adding a link on the home page